.expeditor/config.yml

```diff
@@ -15,17 +15,19 @@ github:
   major_bump_labels:
     - "Expeditor: Bump Version Major"
 
-# https://expeditor.chef.io/docs/getting-started/subscriptions/#merge-actions
-# The `merge_actions` key is a shortcut for the
-# `pull_request_merged:<GITHUB_REPO>:<RELEASE_BRANCH>:*` subscription.
-merge_actions:
-  - built_in:bump_version:
-      ignore_labels:
-        - "Expeditor: Skip Version Bump"
-        - "Expeditor: Skip All"
-  - bash:.expeditor/update_version.sh:
-      only_if: built_in:bump_version
-  - built_in:update_changelog:
-      ignore_labels:
-        - "Expeditor: Skip Changelog"
-        - "Expeditor: Skip All"
+subscriptions:
+  # https://expeditor.chef.io/docs/getting-started/subscriptions/#merge-actions
```

Comment:
We should remove this comment as its no longer relevant.

Reply:
sure